apps/pom.xml

```diff
@@ -162,11 +162,11 @@
 				child projects use, as incompatibility problems with Spring are common. -->
 			<dependency>
 				<!-- Lock & harmonize the versions of the various Spring Framework modules. 
-					We look to HAPI for a guide on what release series to use: ca.uhn.hapi.fhir:hapi-fhir:pom:4.1 
-					uses the 4.3.29.RELEASE version of the Spring Framework. -->
+					We look to HAPI for a guide on what release series to use: ca.uhn.hapi.fhir:hapi-fhir-base:pom:5.7.1
+					uses the 5.3.15 version of the Spring Framework. -->
 				<groupId>org.springframework</groupId>
 				<artifactId>spring-framework-bom</artifactId>
-				<version>4.3.29.RELEASE</version>
+				<version>5.3.15</version>
```

Comment:
reminder - read through spring release notes

Reply:
https://github.com/spring-projects/spring-framework/releases?page=9 (plus the previous 8 pages, ugh)